sdk/iotcentral/test-resources.json

```diff
@@ -0,0 +1,6 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
```

Comment:
remove this file as it's only for test that need prerequisites env

Reply:
removed